### PR TITLE
[merged] Dbus verify

### PR DESCRIPTION
**Pull request by baude**
Atomic/verify.py: Fix dbus implementation of image verify
    
The dbus verify is now similar to the tty version in how it
processes the request.  We now return a list of layers and
their comparisons.

### CODE REVIEW

**Comment by baude** [comment]
I'll squash the wip and last commit if the PR looks good.

File: Atomic/verify.py [hunk 1]
```diff
@@ -74,16 +74,18 @@ def verify(self):
             util.write_out("\n")
 
     def verify_dbus(self):
+        print(self.args)
```
Comment (rhatdan) [inline Atomic/verify.py]: Remove or add if debug.

**Comment by baude** [comment]
atomic image verify as it is currently coded is only one image at a time.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**